sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -51,7 +51,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-sdk-bom</artifactId>
-        <version>1.2.12</version>   <!-- NOTE: This should be updated manually. -->
+        <version>1.2.13</version>   <!-- NOTE: This should be updated manually. -->
```

Comment:
please help check whether the following are already in the azure-sdk-bom: `azure-identity`, `azure-identity-extensions`